README.md

```diff
@@ -29,7 +29,9 @@ The backend is loaded only when chosen. The unused one does not have to be insta
 It splits punctuated text to sentences by full stops, avoiding the dots that are not full stops. The segmenters are language specific.
 The unused one does not have to be installed. We integrate the following segmenters, but suggestions for better alternatives are welcome.
 
-- `pip install opus-fast-mosestokenizer` for the languages with codes `as bn ca cs de el en es et fi fr ga gu hi hu is it kn lt lv ml mni mr nl or pa pl pt ro ru sk sl sv ta te yue zh`
+- for the languages with codes `as bn ca cs de el en es et fi fr ga gu hi hu is it kn lt lv ml mni mr nl or pa pl pt ro ru sk sl sv ta te yue zh`
+- - Apple Silicon: `mosestokenizer` 
```

Comment:
Yes, we need to write about this version, but this is too brief. Can you expand what Apple Silicon means?

Reply:
I think he means the M1/M2 chips.